Mathlib/Topology/Algebra/Valued/WithVal.lean

```diff
@@ -84,8 +84,12 @@ variable [Ring R] (v : Valuation R Γ₀)
 /-- Canonical ring equivalence between `WithVal v` and `R`. -/
 def equiv : WithVal v ≃+* R := RingEquiv.refl _
 
+@[simp]
 theorem apply_equiv (r : WithVal v) : v (WithVal.equiv v r) = v r := rfl
 
+@[simp]
+theorem apply_symm_equiv (r : R) : v ((WithVal.equiv v).symm r) = v r := rfl
```

Comment:
Shouldn't this be:
```suggestion
theorem apply_symm_equiv (r : R) : Valued.v ((WithVal.equiv v).symm r) = v r := rfl
```

Reply:
But `v ...` abuses defeq while `Valued.v ...` doesn't. `(WithVal.equiv v).symm r` has type `WithVal v` which is the input `Valued.v` takes, but `v` takes input of type `R`.